Error message on startup after "grails clean"

Error creating bean with name 'sessionFactory': Invocation of init method failed; nested exception is org.hibernate.MappingException: Missing type or column for column[products_product] on domain[ProductCategory] referencing[de.dewarim.goblin.pc.crafting.Product]
        at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
        at java.util.concurrent.FutureTask.run(FutureTask.java:166)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:724)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'transactionManager': Cannot resolve reference to bean 'sessionFactory' while setting bean property 'sessionFactory'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sessionFactory': Invocation of init method failed; nested exception is org.hibernate.MappingException: Missing type or column for column[products_product] on domain[ProductCategory] referencing[de.dewarim.goblin.pc.crafting.Product]
        ... 5 more
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sessionFactory': Invocation of init method failed; nested exception is org.hibernate.MappingException: Missing type or column for column[products_product] on domain[ProductCategory] referencing[de.dewarim.goblin.pc.crafting.Product]
        ... 5 more
Caused by: org.hibernate.MappingException: Missing type or column for column[products_product] on domain[ProductCategory] referencing[de.dewarim.goblin.pc.crafting.Product]
        ... 5 more

--- a/grails-app/domain/de/dewarim/goblin/UserAccount.groovy
+++ b/grails-app/domain/de/dewarim/goblin/UserAccount.groovy
@@ -8,7 +8,8 @@ import de.dewarim.goblin.pc.PlayerCharacter
 class UserAccount {
 	static transients = ['pass']
 	static hasMany = [userRoles: UserRole,characters:PlayerCharacter]
-
+    static mappedBy = [characters: 'user']
+    
     def springSecurityService
 
 	PlayerCharacter currentChar

--- a/grails-app/domain/de/dewarim/goblin/pc/crafting/ProductCategory.groovy
+++ b/grails-app/domain/de/dewarim/goblin/pc/crafting/ProductCategory.groovy
@@ -6,7 +6,7 @@ package de.dewarim.goblin.pc.crafting
 class ProductCategory {
 
     static hasMany = [products:Product]
-
+    static mappedBy = [products: 'category']
     static constraints = {
         description nullable:true
         name unique: true

--- a/grails-app/domain/de/dewarim/goblin/quest/QuestStep.groovy
+++ b/grails-app/domain/de/dewarim/goblin/quest/QuestStep.groovy
@@ -2,8 +2,7 @@ package de.dewarim.goblin.quest
 
 class QuestStep {
 
-    static belongsTo = [encounter: Encounter, questTemplate: QuestTemplate
-    ]
+    static belongsTo = [encounter: Encounter, questTemplate: QuestTemplate]
     static hasMany = [nextSteps: StepChild, parentSteps: StepChild]
     static mappedBy = [parentSteps: 'child', nextSteps: 'parent']
     static constraints = {

--- a/grails-app/domain/de/dewarim/goblin/quest/QuestTemplate.groovy
+++ b/grails-app/domain/de/dewarim/goblin/quest/QuestTemplate.groovy
@@ -3,6 +3,7 @@ package de.dewarim.goblin.quest
 class QuestTemplate {
 
 	static hasMany = [requirements:QuestRequirement, steps:QuestStep]
+    static mappedBy = [steps:'questTemplate']
     static belongsTo = [giver:QuestGiver]
 	static constraints = {
 		name blank:false

